.markdownlint.yaml

```diff
@@ -0,0 +1,37 @@
+# Rules ref: https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md
+
```

Comment:
can we move this to a common folder?

Reply:
This is in the project root just like the rest of the linter configs.